web/packages/operations/src/alarm.ts

```diff
@@ -157,41 +123,30 @@ export const sendMetrics = async (metrics: status.AllMetrics) => {
             ],
             Value: channel.toEthereum.inbound,
         })
-        metricData.push({
-            MetricName: "ToEthereumPreviousInboundNonce",
-            Dimensions: [
-                {
-                    Name: "ChannelName",
-                    Value: channel.name,
-                },
-            ],
-            Value: channel.toEthereum.previousInbound,
-        })
-        metricData.push({
-            MetricName: "ToEthereumUndelivered",
-            Dimensions: [
-                {
-                    Name: "ChannelName",
-                    Value: channel.name,
-                },
-            ],
-            Value: channel.toEthereum.outbound - channel.toEthereum.inbound,
-        })
-        metricData.push({
-            MetricName: AlarmReason.ToEthereumChannelStale.toString(),
-            Value: Number(
-                channel.toEthereum.outbound > channel.toEthereum.inbound &&
-                    channel.toEthereum.inbound == channel.toEthereum.previousInbound
-            ),
-        })
-        metricData.push({
-            MetricName: AlarmReason.ToEthereumChannelAttacked.toString(),
-            Value: Number(channel.toEthereum.outbound < channel.toEthereum.inbound),
-        })
-        metricData.push({
-            MetricName: AlarmReason.ToEthereumNoTransfer.toString(),
-            Value: Number(channel.toEthereum.inbound == channel.toEthereum.previousInbound),
-        })
+        if (channel.toEthereum.estimatedDeliveryTime) {
+            metricData.push({
+                MetricName: "ToEthereumDeliveryEstimate",
+                Dimensions: [
+                    {
+                        Name: "ChannelName",
+                        Value: channel.name,
+                    },
+                ],
+                Value: channel.toEthereum.estimatedDeliveryTime,
+            })
+        }
+        if (channel.toEthereum.undeliveredTimeout) {
```

Comment:
There is probably a bug. This `if` statement means that if there is no underliveredTimeout then it does not log. Which would produce what we are seeing in prod where the hearbeat is running but this metric is not receiving any data.

Reply:
Maybe if this field is null we should log max value or the threshold so we trip the alarm with a valid value.

Reply:
For the subsquid query, if there is no undeliveredTimeout,  it indicates a normal condition where the nonce matched.

The root cause of the failure to catch it was that I missed adding the Dimension field when initializing the alarm.

https://github.com/Snowfork/snowbridge/pull/1538/commits/735aa3f7efe63cacf4385bdabe608570018264cd